barretenberg/cpp/src/barretenberg/vm2/constraining/relations/range_check.test.cpp

```diff
@@ -0,0 +1,162 @@
+#include <cmath>
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <cstdint>
+
+#include "barretenberg/vm2/constraining/testing/check_relation.hpp"
+#include "barretenberg/vm2/generated/flavor_settings.hpp"
+#include "barretenberg/vm2/generated/relations/range_check.hpp"
+#include "barretenberg/vm2/testing/macros.hpp"
+#include "barretenberg/vm2/tracegen/test_trace_container.hpp"
+
+namespace bb::avm2::constraining {
+namespace {
+
+using tracegen::TestTraceContainer;
+using FF = AvmFlavorSettings::FF;
+using C = Column;
+using range_check = bb::avm2::range_check<FF>;
+
+TEST(AvmConstrainingTest, RangeCheckPositiveIsLteMutuallyExclusive)
```

Comment:
Could you add a test where you
* create a single row with `clk = 1`. this is to force a row to be in the trace. I actually want a completely empty row but it doesnt matter.
* check the whole relation and expect no failures
?

This is to test that all relations pass in rows fulls of 0. We probably need to add this test to every gadget.